package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@fiverr/uploader",
```

Comment:
if it's public maybe it shouldn't be under `@fiverr` prefix (see other packages)